apps/native/app/src/screens/applications/applications-incomplete.tsx

```diff
@@ -0,0 +1,62 @@
+import { useState } from 'react'
+
+import { NavigationFunctionComponent } from 'react-native-navigation'
+import {
+  ApplicationResponseDtoStatusEnum,
+  useListApplicationsQuery,
+} from '../../graphql/types/schema'
+import { createNavigationOptionHooks } from '../../hooks/create-navigation-option-hooks'
+import { useConnectivityIndicator } from '../../hooks/use-connectivity-indicator'
+import { ApplicationsList } from './components/applications-list'
+
+const { useNavigationOptions, getNavigationOptions } =
+  createNavigationOptionHooks(
+    (theme, intl) => ({
+      topBar: {
+        title: {
+          text: intl.formatMessage({ id: 'applications.incomplete' }),
+        },
+        noBorder: true,
+      },
+    }),
+    {
+      bottomTabs: {
+        visible: false,
+        drawBehind: true,
+      },
+    },
+  )
+
+export const ApplicationsIncompleteScreen: NavigationFunctionComponent = ({
+  componentId,
+}) => {
+  useNavigationOptions(componentId)
+  const [refetching, setRefetching] = useState(false)
+
+  const applicationsRes = useListApplicationsQuery({
+    variables: {
+      input: {
+        status: [ApplicationResponseDtoStatusEnum.Draft],
```

Comment:
@snaerseljan One caveat here. In `applications.ts` I sort the applications to show the first four, and there for the incomplete once  I check if `application.**state**` is either `'draft'` or `'prerequisites'` and then it is a part of the incomplete ones. 

The input for the query only allows filtering by **status** so in theory we could be missing some of the ones we are showing on the overview page since we cannot catch the ones that have state = prerequisites (or draft but from what I have seen then both status and state are always draft so we might get them). 

The code I am referring to is [here](https://github.com/island-is/island.is/blob/f885cca1ec07bdd218410e7a95cb5feff5646e15/libs/service-portal/applications/src/shared/utils/index.ts#L27), but it is 2 years old it seems - so I suggest to check with Þórður from Hugsmiðjan if we need to add this filtering to the graphql endpoint to make sure we get everything or if we can change this filtering to just use status 😄  Please let me know if something is unclear about this 🤝

Reply:
I've talked to Norda and this is not relevant anymore, I should be using status to filter and only check for draft status ✅ Will fix 😄